packages/gestalt/src/Contents.js

```diff
@@ -497,6 +497,7 @@ export default class Contents extends React.Component<Props, State> {
     return (
       <div
         className={styles.container}
+        // $FlowFixMe: new errors found from flow 0.116.1 upgrade
```

Comment:
@jennyscript instead of adding these `$FlowFixMe`s, could we fix the errors instead?